plugins/woocommerce/src/Admin/Features/ProductBlockEditor/Init.php

```diff
@@ -42,18 +43,18 @@ class Init {
 	 */
 	public function __construct() {
 		if ( Features::is_enabled( 'product-variation-management' ) ) {
-			array_push( $this->supported_post_types, 'variable' );
+			array_push( $this->supported_product_types, 'variable' );
```

Comment:
FYI, you can just do:

```php
$this->supported_product_types[] = 'variable';
```

Either works!

Reply:
Yep it was just a copy and paste from the other similar implementations.